api/envoy/extensions/filters/http/aws_lambda/v3/aws_lambda.proto

```diff
@@ -62,6 +62,24 @@ message Config {
   // the provider chain is limited to the AWS credentials file Provider.
   // Other providers are ignored
   string credentials_profile = 5;
+
+  // Specifies the credentials to be used. This parameter is optional and if it is set,
+  // it will override other providers and will take precedence over credentials_profile.
+  // The provider chain is limited to the configuration credentials provider.
+  // Default providers chain specified in documentation will be ignored.
```

Comment:
Not sure I fully understand. Do you mean to say:
`If this field is provided, then the default providers chain specified in documentation will be ignored.` ?

Can you please elaborate what documentation is referred here? Consider adding a link.